### PR TITLE
Correct transparent damage graphs in SDL2

### PR DESCRIPTION
**Pull request by Wedge009**
Alpha channel was set to 0, which means that these graphs are not displayed under SDL2. Changed alpha values to 255 to restore the graphs.

SDL1.2 builds are unaffected because they (at least going by the struct definition) ignore the value fourth colour channel.

Other changes are just white-space formatting.

### CODE REVIEW

**Comment by Wedge009** [comment]
**tl;dr**: As discussed with Aginor, this can be merged with the sdl2 branch, no further changes needed at this point.

To comment on other instances of unused/alpha channel hard-coded as 0:

marked-up_text.cpp seems to be okay - can still see coloured text like XP and HP just fine.
Similarly, unit.cpp holds colours for XP and HP coloured text.

sdl/image.cpp is only used if SDL_GPU is defined so not relevant at the moment.
Similarly, SDL_gpu directory isn't currently used either.

Lastly, the progressbar.cpp instance is used in progress bars such as the connecting to MP servers. I haven't noticed anything amiss with SDL2 there either.

**Comment by aginor** [comment]
Manually pushed into sdl2 branch.
